data.json

```
@@ -71,6 +71,15 @@
     "dateAdded": "2018-10-06T03:35:20.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "Blockstack",
+    "difficulty": "medium",
+    "description": "Complete the Zero-to-DApp tutorial and submit your sample to App.co to win a T-shirt!",
```

```suggestion
    "description": "Complete the Zero-to-DApp tutorial and submit your sample to App.co to get a T-shirt!",
```

`Win` makes it feel like a competition